extras/axelar-token/Makefile

```diff
@@ -0,0 +1,49 @@
+include .env
+export
+
+.PHONY: deps
+deps:
```

Comment:
Interesting. The intention is you'd pull and initialize recursive submodules, then you wouldn't need to run install. In fact, I think this concrete `forge install` may be the wrong thing to run here since we probably want to lock dependency versions.

Reply:
I think I've fixed this in [`1723cdb` (#711)](https://github.com/consensus-shipyard/ipc/pull/711/commits/1723cdbbb7af7298fe64e0f5950594d45661500d).